optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/main/resources/org/optaplanner/workbench/screens/solver/backend/server/solution/optacloud.xml

```diff
@@ -1,48 +1,48 @@
-<opta.optacloud.CloudSolution id="1">
+<optacloud.optacloud.CloudSolution id="1">
```

Comment:
Why do we have this file here (as opposed to kie-wb-playground)?
Might be worth adding a one-line comment above for motivation. If this is a temporary thing, make it a TODO